opentelemetry/src/logs/record.rs

```diff
@@ -3,6 +3,11 @@ use std::{borrow::Cow, collections::HashMap, time::SystemTime};
 
 /// SDK implemented trait for managing log records
 pub trait LogRecord {
+    /// Sets the `event_name` of a record
+    fn set_event_name<T>(&mut self, name: T)
```

Comment:
can we provide a {} empty implementation, so SDKs need not implement it, unless they care about it.

Reply:
Good point. done.